src/styles/components/_Header.scss

```diff
@@ -1,13 +1,5 @@
-// The "inverse" header has a blue background and white text
-.hc-c-header--inverse {
-  background-color: $color-primary;
-  color: $color-white;
-
-  .hc-c-logo__health,
-  .hc-c-logo__care,
-  .hc-c-logo__gov {
-    fill: $color-white;
-  }
```

Comment:
Updated the styles for the header as well as cleaned it up. There is still more validation and clean up to do.